moved/src/types/session_id.rs

```diff
@@ -29,6 +29,7 @@ impl SessionId {
         maybe_entry_fn: Option<&EntryFunction>,
         tx_hash: &B256,
         genesis_config: &GenesisConfig,
+        script_hash: Option<[u8; 32]>,
```

Comment:
Should we not use `Option<B256>` similar to `tx_hash`?

Reply:
It means having an extra `map` involved because the aptos library where this ultimately gets used does not know about the alloy types, but I agree being consistent makes sense. Fixed in https://github.com/MovedNetwork/op-move/pull/144/commits/df13715d3eb74708cbd946ab567b3a3b19ff4cd8